README.md

```diff
@@ -115,159 +122,149 @@ _See documentation [here](#tensorflow-and-keras)._
+When the training code is instrumented with Aim SDK's [Python Library](#python-library) and ran, Aim creates the `.aim` repository in your specified path and stores the data. Otherwise the data is created and stored in working directory.
 
-```py
-...
-aim.track(metric_val, name='metric_name', epoch=current_epoch)
-aim.set_params(hyperparam_dict, name='dict_name')
-...
-```
+Additionally, Aim SDK also gives you flexibility to:
+
+- use multiple sessions in one training script to store multiple runs at once. When not initialized explicitly, Aim creates a default session.
+- use experiments to group related runs together. An experiment named `default` is created otherwise.
+- use integrations to automate tracking
+
+Jump to [[Getting Started](#getting-started-in-3-steps)] [[SDK Specifications](#sdk-specifications)] [[Use Cases](#use-cases)]
+
+# SDK Specifications
+
+## Session
+
+Use Session to specify custom `.aim` directory or the experiment from the code.
+
+_Class_ aim.**Session**_()<sub>[source](https://github.com/aimhubio/aim/blob/main/aim/sdk/session/session.py)</sub>_
+
+_Parameters_
+
+- **repo** - Full path to parent directory of Aim repo - the `.aim` directory. By default current working directory.
+- **experiment** - A name of the experiment. By default `default`. See [concepts](#concepts)
+- **flush_frequency** - The frequency per step to flush intermediate aggregated values of metrics to disk. By default per `128` step.
+- **block_termination** - If set to `True` process will wait until all tasks are completed, otherwise pending tasks will be killed at process exit. By default `True`.
+- **run** - A name of the run. If run name is not specified, universally unique identifier will be generated.
+
+_Returns_
+
+- Session object to attribute recorded training run to.
+
+_Methods_
+
+- [`track()`](#track) - Tracks metrics within the session
+
+- [`set_params()`](#set_params) - Sets session params
+
+- [`flush()`](#flush) - Flushes intermediate aggregated metrics to disk. This method is called at a given frequency and at the end of the run automatically.
+
+- `close()` - Closes the session. If not invoked, the session will be automatically closed when the training is done.
+
+_Example_
+
+- [Here](https://github.com/aimhubio/aim/tree/main/examples/sessions) are a few examples of how to use the `aim.Session` in code.
 
 ### track
 aim.**track**_(value, name='metric_name' [, epoch=epoch] [, **context_args]) <sub>[source](https://github.com/aimhubio/aim/blob/6ef09d8d77c517728978703764fc9ffe323f12b0/aim/sdk/track.py#L6)</sub>_
 
 _Parameters_
+
 - **value** - the metric value of type `int`/`float` to track/log
 - **name** - the name of the metric of type `str` to track/log (preferred divider: `snake_case`)
 - **epoch** - an optional value of the epoch being tracked
 - **context_args** - any set of other parameters passed would be considered as key-value context for metrics
 
```

Comment:
Maybe add a note here:
`Note: using aim.track means using the default initiated aim Session`